plugins/modules/junos_static_routes.py

```diff
@@ -136,18 +135,42 @@
     - address_families:
       - afi: ipv4
         routes:
-        - dest: 10.200.16.75/24
+        - dest: 192.168.16.0/24
           next_hop:
-          - forward_router_address: 10.200.16.2
+          - forward_router_address: 172.16.1.2
     state: deleted
 
+# Task Output
```

Comment:
```suggestion
# Task Output:
--------------
```

Reply:
Lets follow the same format for before state and After state as we are doing for all the documentations now:
# Before state:
# -----------------